pkg/controller/volume/persistentvolume/scheduler_bind_cache_metrics.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolume
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+// VolumeSchedulerSubsystem - subsystem name used by scheduler
+const VolumeSchedulerSubsystem = "volumeScheduler"
+
+var (
+	VolumeBindingAddToSchedulerBinderCache = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: VolumeSchedulerSubsystem,
+			Name:      "volume_add_to_scheduler_binder_cache_total",
+			Help:      "Total number for volume binding add to cache",
+		},
+	)
+	VolumeBindingDeleteFromSchedulerBinderCache = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: VolumeSchedulerSubsystem,
+			Name:      "volume_delete_from_scheduler_binder_cache_total",
+			Help:      "Total number for volume binding delete from cache",
+		},
+	)
+	VolumeBindingPredicateLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Subsystem: VolumeSchedulerSubsystem,
+			Name:      "volume_binding_latency_microseconds",
```

Comment:
Also according to the naming best practices, we should be using the base unit of "seconds" instead of "microseconds".

Reply:
I do not think so, what is level for the latency for volume binding and other operation, second level? or microsecond level? And there are many metric name end up with microseconds in k8s code.

Reply:
Prometheus still stores the values with enough precision to be able to convert to units smaller than seconds, but they recommend to standardize on seconds, and then let the display handle conversion of units.

It seems like the units are inconsistent in Kubernetes, and even the scheduler itself uses both seconds and microseconds.  @bsalamat what would you recommend the base unit for scheduler latency metrics be going forward?

Reply:
I know scheduler has other metrics that use "microseconds" and have other naming issues, but for new metrics, we should try to name them properly. So, the unit for time should be seconds.